schema/queries.proto

```diff
@@ -47,13 +51,14 @@ message Query {
        GetSignatories get_account_signatories = 4;
        GetAccountTransactions get_account_transactions = 5;
        GetAccountAssetTransactions get_account_asset_transactions = 6;
-       GetAccountAssets get_account_assets = 7;
-       GetRoles get_roles = 8;
-       GetAssetInfo get_asset_info = 9;
-       GetRolePermissions get_role_permissions = 10;
+       GetTransactions get_transactions = 7;
```

Comment:
Is there any specific reason why it's here and not in the end?

Reply:
I inserted here.
```
GetAccountTransactions get_account_transactions = 5;
GetAccountAssetTransactions get_account_asset_transactions = 6;
<---
GetAccountAssets get_account_assets = 7;
```
Because `GetAccountTransactions`, `GetAccountAssetTransactions`, `GetTransactions` are similar query.

Reply:
Ah, I see